source/_patterns/02-molecules/example-widget/index.js

```diff
@@ -9,15 +9,20 @@ import 'molecules/card';
 
 // Module template
 import './_vue-facet-table-vuex.twig';
+import './_vue-facet-table-vue.twig';
 
 // Module styles
 import './_example-widget.scss';
 
-import FacetTable from './src/vuex-example'; // mount only if dom found
+import FacetTable from './src/vue-example'; // mount only if dom found
+import FacetTableVx from './src/vuex-example'; // mount only if dom found
 
 // Render Vue elements as soon as possible
+if (document.getElementById('vue-facet-table-vue')) {
+  new FacetTable().$mount('#vue-facet-table-vue');
```

Comment:
I'm considering a new approach here where we don't even `Vue.extend()` unless the ID is in the DOM. I'll walk you through it.